### ISSUE TITLE
Accessibility: Brightness & Saturation options

### ISSUE THREAD

**Issue by Member1221** [anegostudios/VintageStory-Issues#255]
Right now, the Brightness slider is locked during survival. which is pretty bad accessibility wise.
Would be best to allow the user to change to their preferred brightness so that the game can be viewed on a variety of monitors, and for people with different levels of eyesight.

**Comment by copygirl** [comment]
I agree. Something that also affects legibility on a screen is the light environment around it. I'd like to be able to see things just as easily on a screen in a dark room at night as I do at day when the sun shines into my window.

To not be able to adjust the brightness in a reasonable range, especially when such an option is already there, is a little frustrating.

Purely informational, it **is** possible to adjust gamma levels with a client-side command by typing 
for example `.clientconfig gammaLevel 1.5` into the chat (`1.0` to reset). You might want to try a comma instead of a decimal point if you run into problems - something about localization. It's also possible to set up a [macro](https://youtu.be/hjdrDgktFRU).

**Comment by Member1221** [comment]
Having to use commands or macros is not very accessible.

This is accessibility we're talking about.

Saturation is even worse, on that front you have to modify the shader it self.

Not very user friendly at all.

**Comment by copygirl** [comment]
> Having to use commands or macros is not very accessible.
> This is accessibility we're talking about.

Hence why I said it's purely informational.

> Saturation is even worse, on that front you have to modify the shader itself.

I thought you could adjust the saturation in-game with the sepia slider? Or are you talking about increasing saturation beyond 100% (0% sepia)?

**Comment by Member1221** [comment]
I’m meaning saturation beyond 100% to make the colours stand out more, especially with higher brightness
